packages/components/tag/src/tag.tsx

```diff
@@ -91,18 +91,25 @@ const Tag = (props: TTagProps) => {
           cursor: default;
           min-width: 0;
           display: flex;
-          background-color: ${props.type === 'warning'
+          border-radius: ${designTokens.borderRadius20};
+          background-color: ${props.isDisabled
```

Comment:
I believe the disabled state should override styles equally for `normal` and `warning` types.

@FilPob Could you please confirm? 🙏